examples/monitors/pattern-check/payload.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scope": "tag:prod-us-west",
+    "merge_level": "none",
+    "mail_recipients": "test1@example.org",
+    "triggers": [
+        "slack-trigger2"
+    ],
+    "suppression_window": "12h",
+    "timezone": "Europe/London",
+    "trigger_template": "notify_content:\n    title: \"Edge Delta Pattern-Check Alert\""
```

Comment:
same here. let's not set this so the monitors created with this json uses default template